digitalpublicgoods/openrims.json

```diff
@@ -0,0 +1,95 @@
+{
+  "name": "OpenRIMS",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://www.openrims.org/"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://github.com/MSH/Pharmadex2",
+      "Deployment guide - https://raw.githubusercontent.com/MSH/Pharmadex2/main/documentation/MTaPS%20Pharmadex%202%20-%20Deployment%20Guide.pdf"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "checkNonPIIAccessMechanism": "Yes",
+    "nonPIIAccessMechanism": "The system uses REST API"
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "We comply with eHealth policies in the deployment countries too"
+    ],
+    "adherenceSteps": [
+      "Privacy policy - https://github.com/MSH/Pharmadex2/blob/main/privacyandterms/privacypolicy.pdf",
+      "Terms of Use - https://github.com/MSH/Pharmadex2/blob/main/privacyandterms/termsofuse.pdf"
+    ]
+  },
+  "standards": {
+    "supportStandards": "Yes",
+    "standardsList": [
+      "Anatomical, Therapeutical and Chemical index, International Non-proprietary Name"
+    ],
+    "evidenceStandardSupport": [
+      "https://pharmadex2.msh.org"
```

Comment:
This link also seems to have been updated to: https://openrims.msh.org/landing

Reply:
Resolved in https://github.com/DPGAlliance/publicgoods-candidates/pull/1493/files